services/klatchat_observer/controller.py

```diff
@@ -88,6 +91,10 @@ def __init__(
         }
         self._sio = None
         self.sio_url = config["SIO_URL"]
+        self.server_url = (
+            self.sio_url
+        )  # TODO: temporary solution, consider using better naming convention
```

Comment:
Consider creating an issue to address this if it should be a priority or if it might cause issues

Reply:
> Consider creating an issue to address this if it should be a priority or if it might cause issues

its rather a stub, as it is not planned separate klat server and socket io server, so I better remove TODO